libraries/helpers.rb

```diff
@@ -69,13 +79,17 @@ def jks_opts(prefix)
     end
 
     # Return hash with SSL options for OpenSSL
-    def ssl_opts
+    def cdap_ssl_opts(prefix = 'dashboard')
       ssl = {}
-      ssl['keypath'] = node['cdap']['cdap_security']['dashboard.ssl.key']
-      ssl['certpath'] = node['cdap']['cdap_security']['dashboard.ssl.cert']
+      ssl['keypath'] = node['cdap']['cdap_security']["#{prefix}.ssl.key"]
```

Comment:
We don't want to write a `nil` and these are required for SSL. I will guard it in the invocation code. They cannot be empty if `ssl.enabled`, or it's invalid.